packages/apps/graph/src/services/node-service.ts

```diff
@@ -52,3 +69,21 @@ export async function getAccountDetails(
 
   return result as ChainModuleAccountDetails;
 }
+
+export async function sendRawQuery(code: string, chainId: string) {
+  try {
+    const commandResult = await getClient(chainId as ChainId).dirtyRead(
+      Pact.builder
+        .execution(code)
+        .setMeta({
+          chainId: chainId as ChainId,
+        })
+        .setNetworkId(dotenv.NETWORK_ID)
+        .createTransaction(),
+    );
+
+    return JSON.stringify(commandResult.result);
```

Comment:
Before, for the `pactQuery` we were returning the `commandResult.result.data` stringified and not just the result.
```suggestion
    if (commandResult.result.status !== 'success') {
      throw new PactCommandError(
        'Pact Command failed with error',
        commandResult,
      );
    }
    return JSON.stringify(commandResult.result.data);
```

Reply:
Correct. Though I can imagine that if you want to run raw queries, you would also want to have the raw output (including the info on how the request was built). @alber70g What do you think

Reply:
That's a good point actually

Reply:
Nonetheless, I think we should still throw an error if the status is not success in order to give a more descriptive error message

Reply:
Yep, applied 👍